2015-Fall/kvkunkel/Calculator/Calculator/ViewController.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  ViewController.swift
+//  Calculator
+//
+//  Created by Kristoffer Kunkel on 31/08/15.
+//  Copyright (c) 2015 Kristoffer Kunkel. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController
+{
+    
+    @IBOutlet weak var display: UILabel!
+    
+    @IBOutlet weak var historyDisplay: UILabel!
+
+    var typingNumber: Bool = false
```

Comment:
You can omit the type as it is clear from the context i.e. `var typingNumber = false`